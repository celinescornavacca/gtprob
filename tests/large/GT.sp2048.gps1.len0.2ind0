(((((((((QVB0,RVB0),RUB0),KNB0),KAA0),((TOB0,UOB0),((ETC0,FTC0),FRA0))),(((((NEC0,OEC0),((QLA0,RLA0),(FHA0,GHA0))),(LFA0,(OUA0,(OYC0,PYC0)))),(((((((((((BDC0,CDC0),(((AFB0,BFB0),RXA0),QOA0)),(LNB0,MNB0)),((LXA0,MXA0),(ZXB0,((NMC0,(NWC0,OWC0)),XAC0)))),(ZPC0,AQC0)),(((SSA0,(SUA0,TUA0)),(ORC0,PRC0)),IGA0)),(((GJC0,HJC0),((HCC0,ICC0),POA0)),(((JTA0,(EGB0,FGB0)),((FKC0,GKC0),XTA0)),((JIB0,((GRC0,HRC0),FXB0)),(CFC0,(WUC0,XUC0)))))),((RXB0,((RVC0,SVC0),MJC0)),MHA0)),((((JFA0,((((PVC0,QVC0),((GQB0,HQB0),HZA0)),(NLB0,OLB0)),BGA0)),GEA0),GBA0),((ZBA0,(TXC0,UXC0)),((DCA0,(VQA0,((OZC0,PZC0),OTB0))),(((CQB0,DQB0),(PIC0,QIC0)),IBA0))))),(ZCA0,ADA0)),(((((((RZB0,(SBC0,TBC0)),((DLB0,(DWC0,EWC0)),GPA0)),((LLA0,((ERB0,FRB0),(KTA0,(OWB0,(DBC0,EBC0))))),((EXA0,FXA0),(VXB0,WXB0)))),(XBA0,((AOB0,BOB0),((AHB0,(EYB0,FYB0)),BAB0)))),(BBA0,CBA0)),(QDA0,(LKB0,MKB0))),(((((WBA0,((RTB0,STB0),(WVA0,XVA0))),(((YAC0,ZAC0),(((ALA0,((XQA0,YQA0),UMA0)),(HHC0,IHC0)),((BLB0,CLB0),FHB0))),((TCA0,UCA0),(JXC0,KXC0)))),(((ACA0,(((CRA0,(NBB0,OBB0)),((DTA0,ETA0),NRA0)),NCA0)),((URA0,(VSA0,WSA0)),((QZC0,RZC0),(FLA0,(BVA0,CVA0))))),((RAA0,(VAA0,(MCA0,((CBB0,DBB0),((GTB0,HTB0),DIA0))))),(MAA0,((KIA0,(SPA0,TPA0)),(NGB0,OGB0)))))),AAA0),(((((CGA0,((HCB0,(IFB0,JFB0)),ARA0)),(UBB0,VBB0)),(((PZB0,QZB0),(CSB0,DSB0)),(EFB0,FFB0))),((((((VTC0,WTC0),GXA0),WRA0),(ZJA0,(OSB0,PSB0))),((HWA0,(KKC0,LKC0)),HGA0)),(((BOA0,COA0),((BIB0,(NYB0,(AGC0,BGC0))),BBB0)),PEA0))),((((((VZA0,WZA0),LHA0),((NCB0,OCB0),QFA0)),(((SWB0,TWB0),((CGB0,DGB0),((RTA0,(QYC0,RYC0)),AOA0))),(UIA0,(PUA0,(FHC0,GHC0))))),((OHA0,PHA0),(UPA0,((PXB0,QXB0),((GSB0,HSB0),ROB0))))),((HRA0,(KTB0,LTB0)),(QPA0,(GWB0,HWB0))))))))),(((((((UOC0,VOC0),(XCC0,(ATC0,BTC0))),SLA0),(SNB0,(EFC0,FFC0))),(((XPA0,YPA0),EEA0),(NIA0,(DMA0,(ECC0,FCC0))))),((((XVC0,YVC0),VSB0),UVA0),(EIA0,(PWC0,QWC0)))),(((IZC0,JZC0),((((((UZB0,(JWC0,KWC0)),(GMA0,HMA0)),WFA0),((AVC0,BVC0),KGA0)),((((EAB0,(GAD0,HAD0)),(GAB0,HAB0)),((XAB0,YAB0),(SBB0,TBB0))),((DEB0,EEB0),(YYC0,ZYC0)))),((EJA0,FJA0),((YDB0,ZDB0),LCB0)))),((RJA0,SJA0),HIA0))))),((HAA0,((IAA0,((XOA0,YOA0),ZMA0)),(((FIA0,GIA0),(RMB0,SMB0)),(((AUA0,BUA0),(ISA0,((FAB0,(VUB0,(YYB0,(QDC0,RDC0)))),(ZYB0,AZB0)))),((KTC0,LTC0),(((HMC0,IMC0),((KUC0,LUC0),BZB0)),CHA0)))))),((((((((EJB0,FJB0),(SHB0,THB0)),(((TMA0,(STA0,(TTB0,UTB0))),((LIA0,MIA0),(KNA0,LNA0))),(MIB0,NIB0))),NDA0),(ZDA0,(((QZA0,((NZB0,OZB0),AWB0)),XLA0),(ZJB0,AKB0)))),((BDB0,CDB0),((OMB0,PMB0),(IWA0,JWA0)))),((KIB0,LIB0),(DVA0,EVA0))),(((DIB0,EIB0),(BVB0,((ZKC0,ALC0),(REC0,SEC0)))),((DDC0,(GYC0,HYC0)),((((AAD0,BAD0),IUB0),AJB0),SNA0)))))),(((QHB0,RHB0),(((((((XKA0,((ZLB0,AMB0),HKB0)),(WHB0,(NNB0,ONB0))),((((GJA0,HJA0),(LRB0,MRB0)),(UGB0,(NVC0,OVC0))),((((OHB0,PHB0),LZA0),((VIB0,WIB0),(EGC0,(MIC0,NIC0)))),YBA0))),(((BKB0,CKB0),(((UDB0,VDB0),(UHC0,VHC0)),((KSB0,LSB0),DFB0))),((SCB0,((TSC0,USC0),FDB0)),(XHB0,(LHC0,MHC0))))),((TJA0,(((SXB0,TXB0),ZOA0),(WLA0,(WQA0,(FBB0,GBB0))))),((((((LPB0,MPB0),((BRA0,(QGB0,RGB0)),((LAB0,MAB0),HTA0))),((JGC0,KGC0),UFA0)),((((HXA0,IXA0),(ADB0,(FNB0,GNB0))),(DNA0,(((((JNC0,KNC0),LBB0),KBB0),(FZA0,GZA0)),(TPB0,UPB0)))),(((NHA0,(EJC0,FJC0)),XAA0),((TTA0,(XYA0,(KDC0,LDC0))),(EDA0,FDA0))))),((((((AUB0,BUB0),(FNA0,GNA0)),(IDA0,((MLA0,((DZA0,EZA0),(((((SSB0,TSB0),(SRB0,TRB0)),(ITC0,JTC0)),NAB0),YYA0))),((MMA0,(GQA0,(TLB0,((RPC0,SPC0),HFC0)))),LDA0)))),((((IVC0,(FXC0,GXC0)),(OIB0,PIB0)),((TIB0,UIB0),(((NNC0,(LQC0,MQC0)),CNC0),DDB0))),NMA0)),PAA0),(VCC0,WCC0))),((WNA0,((XGC0,YGC0),((WWB0,XWB0),(BBC0,CBC0)))),(((NPA0,(XFC0,(GOC0,HOC0))),((WTB0,XTB0),KGB0)),ZKA0))))),(((MUC0,NUC0),(WDA0,((((AYB0,BYB0),HEB0),CUA0),(ZPA0,AQA0)))),(PPC0,QPC0))),((JVB0,(GIC0,HIC0)),((((TLC0,ULC0),YKC0),LYB0),((VLC0,WLC0),(QOB0,(MYC0,NYC0))))))),((((((((BHB0,(IZB0,(JBC0,KBC0))),HQA0),XMA0),(((((SHC0,THC0),(SXA0,(CIB0,(SFC0,TFC0)))),((YGB0,ZGB0),((ZXA0,AYA0),(JRA0,(XBB0,(JLC0,KLC0)))))),((((TMB0,(VNC0,WNC0)),FOA0),(DUA0,(HXC0,IXC0))),VBA0)),(((SWA0,TWA0),((((JLB0,KLB0),NEB0),(MSB0,NSB0)),((ROA0,SOA0),((NQB0,OQB0),(AZC0,BZC0))))),((BNA0,CNA0),GKA0)))),((((((IJB0,JJB0),(((MDC0,NDC0),EYA0),FWA0)),(MOC0,NOC0)),(((DNB0,ENB0),AGA0),(((YIB0,ZIB0),(EQA0,(KPB0,(BQC0,CQC0)))),VFA0))),(((RCA0,SCA0),(NLA0,OLA0)),((RZA0,SZA0),((((((ZOB0,APB0),GDB0),OQA0),(FVA0,GVA0)),((((DLC0,ELC0),GFC0),ZFB0),(ISC0,(UUC0,VUC0)))),CKA0)))),(PCA0,(((CVB0,DVB0),ETB0),((UHB0,(UQB0,VQB0)),EKA0))))),((((LXB0,MXB0),((ACC0,BCC0),(IIA0,(VAC0,WAC0)))),(((((FTA0,GTA0),((UKA0,((FIB0,GIB0),(CPB0,(OFC0,PFC0)))),((WEC0,XEC0),(VRA0,(EMB0,FMB0))))),((GDA0,(VDA0,(((PYB0,QYB0),(PLB0,QLB0)),(((JMC0,KMC0),LJC0),(MYB0,(WIC0,XIC0)))))),(((LPC0,MPC0),IZA0),(OXA0,PXA0)))),(ZTC0,AUC0)),((EPA0,FPA0),(UOA0,VOA0)))),(((((((TKB0,UKB0),YNA0),(((LLC0,MLC0),HVA0),ZQA0)),(((((BRC0,CRC0),IWB0),BYA0),((KYA0,LYA0),KLA0)),((ZTA0,((LBC0,MBC0),(HFB0,(DPB0,EPB0)))),VHA0))),(((((JPA0,(DUB0,EUB0)),((IOA0,JOA0),XNA0)),((JZA0,KZA0),(RIC0,SIC0))),XDA0),(((DUC0,EUC0),IYB0),(KFA0,(YMB0,(OOC0,POC0)))))),(((((QBB0,((YRB0,ZRB0),PDB0)),((QKB0,(GZB0,HZB0)),(QNB0,RNB0))),((NGA0,(LJA0,((QWB0,RWB0),((FUB0,(OVB0,PVB0)),NQA0)))),(YCA0,(YKA0,(((OTC0,PTC0),WVB0),(FPC0,GPC0)))))),(((KDA0,(WOB0,(DEC0,EEC0))),(AEA0,(LMA0,(((CHB0,DHB0),((UBC0,VBC0),(OOB0,(KXB0,(VJC0,WJC0))))),UQA0)))),GAA0)),(JCB0,KCB0))),(((AVB0,((SZC0,TZC0),IVB0)),JGB0),(((((ZWC0,AXC0),(NJA0,(LKA0,(DPA0,(PEB0,QEB0))))),NAA0),(((((FLC0,GLC0),(QUC0,RUC0)),(YCB0,ZCB0)),RBA0),(COB0,(((XRC0,YRC0),XZB0),SOB0)))),((((GLA0,HLA0),YDA0),(((((AIA0,BIA0),(((KYC0,LYC0),(FGA0,GGA0)),(WGB0,(VKB0,WKB0)))),(((((KFC0,LFC0),IOB0),FQA0),(TPC0,UPC0)),((CAB0,DAB0),VVA0))),((LMB0,MMB0),VEA0)),(((((((UKC0,VKC0),NWB0),(CEC0,(ZOC0,APC0))),BEB0),((SQA0,TQA0),(ZUA0,AVA0))),((AKA0,(RRA0,SRA0)),(TGA0,((GDC0,HDC0),CCC0)))),(((WBB0,(EKB0,(ODC0,PDC0))),((RJC0,SJC0),APA0)),(QAB0,RAB0))))),(BMC0,CMC0))))))),((((JCA0,((((NPB0,(OPB0,(OAC0,PAC0))),(URB0,VRB0)),(JEC0,KEC0)),(XIB0,(LMC0,MMC0)))),((QEA0,(WEA0,(((TAC0,UAC0),((LCC0,MCC0),(ICB0,(INB0,JNB0)))),((QSB0,RSB0),(ZRA0,(YKB0,ZKB0)))))),(AZA0,(WFC0,(PQC0,QQC0))))),CAA0),(((((QQA0,(((BXC0,CXC0),DRC0),WBC0)),(KUA0,LUA0)),((JBB0,(YUC0,ZUC0)),ONA0)),YEA0),(ZHA0,((((MTB0,NTB0),BCB0),TKA0),BKA0))))),((((PDA0,(((XUB0,(IUC0,JUC0)),JPB0),(((VMA0,WMA0),(VRC0,WRC0)),RFA0))),(KRC0,LRC0)),(FUC0,(NXC0,OXC0))),(((((((HMB0,IMB0),BMA0),((((AFC0,BFC0),((OXB0,(IKC0,JKC0)),(DNC0,ENC0))),((QVA0,RVA0),FMA0)),(PSC0,QSC0))),(((((LUB0,MUB0),(HPC0,IPC0)),(((IKB0,JKB0),((TRC0,URC0),QPB0)),OIA0)),((BSC0,CSC0),((KPA0,LPA0),BJA0))),(CXA0,DXA0))),((((((LBA0,((ESB0,FSB0),HEA0)),((WKA0,((CAC0,DAC0),((TWC0,UWC0),(HGB0,IGB0)))),(KMA0,(TXA0,((UXB0,(NSC0,OSC0)),WCB0))))),(VCA0,WCA0)),(((MRC0,NRC0),BIC0),((XEB0,(VPC0,WPC0)),(IMA0,JMA0)))),((((EHA0,(CYA0,DYA0)),(EUA0,FUA0)),(((BPA0,CPA0),((XZA0,YZA0),CZA0)),(FSC0,GSC0))),(((((MDA0,(GFB0,(MHB0,NHB0))),((FEA0,(GSA0,(((SAC0,(NLC0,OLC0)),(GHB0,(VHB0,(XXB0,YXB0)))),(KJB0,LJB0)))),((IXB0,((QOC0,ROC0),(IYC0,JYC0))),RSA0))),QBA0),((((GYB0,HYB0),PBB0),((DWA0,EWA0),(FCB0,GCB0))),((BTA0,CTA0),(QFB0,RFB0)))),(((OCA0,((HAC0,IAC0),(((GWA0,(VXA0,WXA0)),((DYC0,EYC0),(TTC0,UTC0))),QGA0))),(NDB0,(RIB0,SIB0))),((DIC0,(CZC0,DZC0)),(ZFA0,(IAB0,((RKB0,SKB0),XJB0)))))))),(((((WMC0,XMC0),(KHA0,((ZRC0,ASC0),AAC0))),UAA0),(((((((PAB0,(MJB0,NJB0)),(XJC0,YJC0)),((EWB0,FWB0),IRA0)),(DSA0,ESA0)),(MFA0,(FEC0,GEC0))),(UJA0,((SLB0,(VSC0,WSC0)),((YZB0,ZZB0),HHB0)))),((JYB0,KYB0),(OKC0,PKC0)))),((PGB0,(YMC0,ZMC0)),(((PGA0,(HNA0,INA0)),(((YPB0,ZPB0),HSA0),((XPC0,YPC0),VTB0))),((((((EDB0,((ZQC0,ARC0),YCC0)),WOA0),(YWA0,ZWA0)),((DOA0,EOA0),(((AEB0,(WUB0,(DMC0,EMC0))),(UFB0,VFB0)),(HRB0,IRB0)))),((GUB0,HUB0),(UZC0,VZC0))),((((QKC0,RKC0),GCC0),XDB0),(HPB0,IPB0)))))))),((((((((RXC0,SXC0),((((((((JVC0,KVC0),QNC0),FZB0),((SMC0,TMC0),(LOA0,(XXA0,YXA0)))),(((FYC0,(KZC0,LZC0)),ZBB0),(ZLC0,AMC0))),((((VTA0,WTA0),(QNA0,RNA0)),(((UXA0,((ZXC0,AYC0),NVB0)),((KWA0,((IQB0,JQB0),QWA0)),MKA0)),REA0)),(((BUC0,CUC0),(JAC0,KAC0)),(VAB0,WAB0)))),((((IOC0,JOC0),((SKC0,TKC0),WDC0)),(((YFC0,ZFC0),(CYB0,DYB0)),HKA0)),(((((UEA0,(JIA0,(PRA0,QRA0))),(OSA0,((BWC0,CWC0),((YSC0,ZSC0),((BPC0,CPC0),CIC0))))),((GOA0,((ULB0,VLB0),SFB0)),TNA0)),(NBA0,(((FQC0,GQC0),ALB0),(SDB0,TDB0)))),(ZAA0,(((UEB0,(GMB0,(GOB0,HOB0))),WIA0),((FPB0,GPB0),(JDB0,KDB0))))))),(((((ICA0,(RDA0,((((QAD0,RAD0),(MAC0,NAC0)),((CWB0,DWB0),(MRA0,(PWA0,(RCB0,((IHB0,JHB0),LGB0)))))),(SEA0,(DKB0,(PLC0,QLC0)))))),((((((MEA0,(DLA0,(OOA0,(((WQB0,XQB0),((DPC0,EPC0),HDB0)),LRA0)))),((((((LWB0,(IRC0,JRC0)),XSA0),(MUA0,NUA0)),KOA0),(BSA0,CSA0)),EFA0)),MBA0),(ENA0,((WYB0,XYB0),CCB0))),((((ZEA0,(((RWC0,SWC0),(IIC0,JIC0)),(JWB0,KWB0))),((((NCC0,OCC0),JSB0),(QUA0,RUA0)),((DKA0,(FLB0,GLB0)),(OGA0,(((KCC0,(BKC0,CKC0)),((LWA0,MWA0),(SGB0,TGB0))),(KOB0,(SPB0,(IDC0,JDC0)))))))),((((SHA0,THA0),(MKC0,NKC0)),(((XJA0,YJA0),(QSA0,(TSA0,USA0))),(OJA0,(NYA0,OYA0)))),SAA0)),((HYA0,IYA0),((BFA0,((DQC0,EQC0),(EOC0,FOC0))),(JDA0,(((ZYA0,(XQC0,YQC0)),VNA0),(IVA0,((OAD0,PAD0),MGB0)))))))),(((VJA0,WJA0),(ANA0,(TGC0,UGC0))),(((HLC0,ILC0),((UYC0,VYC0),NRB0)),UBA0)))),(((XLB0,YLB0),EHB0),((((GRA0,((ASB0,BSB0),JOB0)),(((CVC0,DVC0),(DTB0,(GVC0,HVC0))),JKA0)),((KHB0,LHB0),((LEC0,MEC0),RBB0))),(((ZGA0,(((LVB0,MVB0),CUB0),ODB0)),(((CTC0,DTC0),((HNB0,(JUB0,KUB0)),UJB0)),WYA0)),((TYA0,(TIC0,UIC0)),(((WHC0,XHC0),YJB0),(PNA0,(UFC0,VFC0)))))))),((((MBB0,(RQC0,SQC0)),(YUA0,(WLB0,(BNB0,CNB0)))),((XDC0,YDC0),MPA0)),BEA0)),(((DXC0,EXC0),((DCB0,ECB0),(JHA0,(HUA0,IUA0)))),(((((QMC0,RMC0),YHC0),(MCB0,(QMB0,(BWB0,(YZC0,ZZC0))))),((LGC0,MGC0),ACB0)),(NTA0,OTA0)))))),DAA0),(MQA0,(AEC0,BEC0))),((TEA0,(NMB0,(ANC0,BNC0))),(RGA0,SGA0))),(((OBA0,PBA0),(((RJB0,((TJC0,UJC0),VOB0)),YHB0),(KRB0,(SOC0,TOC0)))),EAA0)),(((((PTA0,QTA0),PFA0),HDA0),((((XTC0,YTC0),EZB0),(RPA0,(DJB0,(SUB0,(RSC0,SSC0))))),(OYB0,((XLC0,YLC0),ZDC0)))),(((((WRB0,XRB0),((((SZB0,TZB0),(SQB0,TQB0)),((BJB0,CJB0),(TUB0,UUB0))),YLA0)),((YEB0,ZEB0),((BTB0,CTB0),GEB0))),(((OAB0,(IEB0,JEB0)),(YMA0,(HPA0,IPA0))),CIA0)),((BXB0,CXB0),ELA0)))),((ULA0,VLA0),((YEC0,ZEC0),(ASA0,(ZNC0,AOC0)))))),((((((((((((SUC0,TUC0),MYA0),((LDB0,MDB0),((((KAD0,LAD0),HKC0),(HWC0,IWC0)),(((((XGB0,(ORB0,PRB0)),AAB0),(EDC0,FDC0)),KRA0),((ZJC0,AKC0),(((TQC0,UQC0),SGC0),(KFB0,LFB0))))))),((WUA0,XUA0),SFA0)),(XBC0,YBC0)),((LSC0,MSC0),SRC0)),BAA0),(((((((BWA0,(ZHB0,AIB0)),((LQB0,MQB0),(VEC0,(LWC0,MWC0)))),(((RKA0,(FYA0,GYA0)),DEA0),((BDA0,(((PKB0,(WSB0,XSB0)),EMA0),IKA0)),((DGA0,EGA0),(QXA0,(((QHC0,RHC0),LAC0),POB0)))))),(((OAA0,((ZSB0,ATB0),YSA0)),((ECA0,FCA0),TBA0)),((((((YTB0,ZTB0),VGB0),KXA0),(JBA0,((FBC0,GBC0),(CEB0,(VQC0,WQC0))))),(((PQB0,QQB0),PNB0),(QHA0,(MNA0,NNA0)))),(MAD0,NAD0)))),((RIA0,SIA0),HBA0)),(((((((EAC0,FAC0),FTB0),(UYA0,VYA0)),(IJA0,((JAB0,KAB0),(((LNC0,MNC0),(NPC0,OPC0)),(((IAD0,JAD0),KQB0),(PXC0,QXC0)))))),(VZB0,WZB0)),(((DFC0,(FNC0,GNC0)),UAB0),(BLA0,(TVC0,UVC0)))),((((((SVA0,TVA0),XHA0),(((TCB0,(ITB0,JTB0)),NZA0),(BHA0,(((RQB0,(QUB0,((VVC0,WVC0),OIC0))),(DHC0,EHC0)),(OKA0,PKA0))))),(LOB0,(JPC0,KPC0))),(((FKA0,(IQA0,JQA0)),(MEB0,((FMC0,GMC0),(CMB0,DMB0)))),(((CMA0,(NKB0,OKB0)),((XEA0,((PEC0,QEC0),MOA0)),(((ZAB0,ABB0),(OJB0,PJB0)),((ZHC0,AIC0),(EVB0,FVB0))))),LAA0))),((BCA0,(((SKA0,((UUA0,VUA0),(FKB0,GKB0))),LGA0),(DSC0,ESC0))),((BOC0,COC0),ONC0))))),(((((BHC0,CHC0),(NQC0,OQC0)),(((YIA0,ZIA0),(OMA0,((DCC0,(EZC0,FZC0)),(VXC0,WXC0)))),CDA0)),(((SDA0,TDA0),((PPB0,(UWB0,VWB0)),(BMB0,(WPB0,XPB0)))),((((LZB0,MZB0),MLB0),TFA0),(((PCB0,QCB0),((((XKB0,(RYB0,(RCC0,SCC0))),(QFC0,RFC0)),(AQB0,BQB0)),(JVA0,KVA0))),((CJA0,(BLC0,CLC0)),HHA0))))),((PMA0,QMA0),(EOB0,FOB0))))),(((ABA0,((((YSB0,(CJC0,DJC0)),JXA0),FSA0),(QCA0,((QRC0,RRC0),((ITA0,(WKC0,XKC0)),(UMC0,VMC0)))))),((((RNC0,SNC0),ISB0),(ZMB0,((VWC0,WWC0),ABC0))),ODA0)),((((VKA0,((PUB0,(YUB0,ZUB0)),YVA0)),(HOA0,(PVA0,(TZA0,UZA0)))),KBA0),(((NFA0,OFA0),CEA0),(((ZCC0,ADC0),JYA0),XGA0))))),(((((((JSC0,KSC0),CWA0),((SDC0,TDC0),PIA0)),(((PNC0,((CAD0,DAD0),XSC0)),TOA0),(JGA0,((PSA0,(UWA0,VWA0)),NKA0)))),((((XFA0,YFA0),((XIA0,((AXA0,BXA0),(((PJC0,QJC0),ELB0),GJB0))),((HGC0,IGC0),(AXB0,(UYB0,VYB0))))),EBA0),TAA0)),((((UHA0,((IDB0,(RPB0,(QGC0,RGC0))),YBB0)),(SJB0,TJB0)),FAA0),(((HNC0,INC0),(SAB0,TAB0)),((CZB0,DZB0),XCA0)))),(((JAA0,((KEA0,LEA0),((MOB0,NOB0),DOB0))),(((RWA0,(NXB0,(CGC0,DGC0))),(WHA0,((HIB0,IIB0),(ZVA0,AWA0)))),(((((QIB0,((USB0,(SYB0,TYB0)),ANB0)),AJA0),(((DXB0,EXB0),NXA0),JNA0)),(((NEA0,(((GUA0,((RLC0,SLC0),(XVB0,YVB0))),(KOC0,LOC0)),(HBB0,IBB0))),(QAC0,RAC0)),(((((FEB0,(IJC0,JJC0)),EBB0),(MFB0,NFB0)),(RLB0,((GUC0,HUC0),YIC0))),(XNB0,YNB0)))),(LTA0,MTA0)))),((((TRA0,(MSA0,(GGB0,(EIC0,FIC0)))),(OMC0,PMC0)),((((((REB0,(JQC0,KQC0)),(SEB0,TEB0)),((XOB0,YOB0),(LVA0,MVA0))),DDA0),KCA0),(CFA0,DFA0))),(((OZA0,(JZB0,KZB0)),((((YWB0,ZWB0),PWB0),(JRB0,(NGC0,OGC0))),(((ERC0,FRC0),RBC0),(SYC0,TYC0)))),((ZNA0,(CQA0,DQA0)),((((PBC0,QBC0),KEB0),(VGA0,((RMA0,SMA0),DHA0))),(((PCC0,QCC0),ZVB0),(WZC0,XZC0))))))))),(((QAA0,(((TNB0,UNB0),WGA0),((RYA0,SYA0),((JXB0,(FGC0,GGC0)),LCA0)))),(JJA0,KJA0)),((((VGC0,WGC0),(ZBC0,(WOC0,XOC0))),JSA0),NOA0))),((((((HCA0,((((VNB0,WNB0),(NJC0,OJC0)),CLA0),((KIC0,LIC0),((MTC0,NTC0),(AGB0,BGB0))))),((((WWA0,XWA0),((JUA0,(WFB0,XFB0)),((((LXC0,MXC0),KJC0),((TCC0,UCC0),HJB0)),TFB0))),(((((EVC0,FVC0),(DOC0,(LVC0,MVC0))),KKB0),((((MZC0,NZC0),PGC0),(MFC0,NFC0)),(WYC0,XYC0))),(((GAC0,(OUC0,PUC0)),LLB0),UGA0))),((((IFC0,JFC0),(GXB0,HXB0)),(MJA0,(BQA0,(YTA0,(GVB0,HVB0))))),(GCA0,(((DRA0,ERA0),(ZLA0,(ZSA0,ATA0))),OEA0))))),((((NVA0,OVA0),IEA0),(((NBC0,OBC0),((ZZA0,((XNC0,YNC0),JCC0)),VPA0)),((ORA0,(UMB0,VMB0)),(((GTC0,HTC0),CFB0),QIA0)))),(((BJC0,(SAD0,TAD0)),VPB0),PLA0))),((RQA0,((MWB0,(HBC0,IBC0)),LEB0)),(((((((ZVC0,AWC0),YOC0),NHC0),(((QTC0,RTC0),HSC0),(OPA0,PPA0))),MGA0),((NWA0,OWA0),((PTB0,QTB0),PQA0))),(((TEC0,UEC0),XCB0),((BAC0,(EAD0,FAD0)),SBA0))))),((((QDB0,RDB0),QKA0),(DBA0,((HFA0,IFA0),((XRA0,YRA0),CCA0)))),((((XWC0,YWC0),STC0),OEB0),FFA0))),((((KKA0,(ZGC0,AHC0)),AHA0),((WAA0,(((HEC0,IEC0),KVB0),(((HLB0,ILB0),VIA0),(TIA0,(UNA0,(OHC0,PHC0)))))),(JEA0,(CRB0,DRB0)))),(((OFB0,PFB0),(PYA0,QYA0)),((((YQB0,ZQB0),ZNB0),(UDC0,VDC0)),((WMB0,XMB0),(HQC0,IQC0)))))))))))),((YAA0,(UDA0,(WPA0,(SVB0,TVB0)))),((((((VEB0,WEB0),(AFA0,(YGA0,(((JMB0,KMB0),(((UVB0,VVB0),BPB0),(((EQB0,FQB0),((BYC0,CYC0),GRB0)),PZA0))),((JHC0,KHC0),((NSA0,(((DKC0,EKC0),VIC0),UTA0)),AMA0)))))),(IHA0,(QJB0,(TNC0,UNC0)))),((NUB0,OUB0),WDB0)),(FBA0,(((PJA0,QJA0),GFA0),(((KSA0,LSA0),(ARB0,BRB0)),YHA0)))),(((KQA0,LQA0),(((((VJB0,WJB0),MZA0),((QRB0,RRB0),(BZA0,(ZIC0,AJC0)))),RHA0),(((YFB0,(FWC0,GWC0)),(UCB0,VCB0)),(TLA0,(GZC0,HZC0))))),((ILA0,JLA0),(DJA0,(XXC0,YXC0)))))));
