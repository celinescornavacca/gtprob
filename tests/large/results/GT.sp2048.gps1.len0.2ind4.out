-2779.1092096
